apps/lib/apps.c

```diff
@@ -1644,8 +1648,8 @@ X509_NAME *parse_name(const char *cp, long chtype, int canmulti)
             *bp++ = *cp++;
         if (*cp == '\0') {
             BIO_printf(bio_err,
-                       "%s: Hit end of string before finding the '='\n",
-                       opt_getprog());
+                       "%s: Hit end of %d name string before finding the '='\n",
```

Comment:
There ought to be a better word that "Hit".  What about "Reached"?

Reply:
Yes, in this case we can simply (and more concisely) replace "Hit end of" by "End of" - done that.